.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+Thank you for contributing to TiKV! Please read TiKV's [CONTRIBUTING](https://github.com/pingcap/tikv/blob/master/CONTRIBUTING.md) document **BEFORE** filing this PR.
```

Comment:
I think we can put this in an HTML comment so it doesn't show up in the rendered markdown after they submit.

Reply:
@Hoverbear Done, PTAL.